src/sources/logstash.rs

```diff
@@ -144,10 +151,12 @@ impl TcpSourceAcker for LogstashAcker {
     fn build_ack(self, ack: TcpSourceAck) -> Option<Bytes> {
         match ack {
             TcpSourceAck::Ack => {
-                let mut bytes: Vec<u8> = Vec::with_capacity(6);
-                bytes.push(self.protocol.into());
-                bytes.push(LogstashFrameType::Ack.into());
-                bytes.extend(self.sequence_number.to_be_bytes().iter());
+                let mut bytes: Vec<u8> = Vec::with_capacity(6 * self.sequence_numbers.len());
```

Comment:
Oh hmm, why 6 in the first place?

Reply:
I believe 6 bytes in for protocol (`u8`) + frame type (`u8`) + sequence number (`u32`), so a known fixed size of an ack frame in this protocol.

Reply:
Ah, gotcha. Thank you.

Reply:
A future reader would probably appreciate having a constant along the lines of

```rust
const ACK_FRAME_SIZE: usize = 6; // protocol (u8) + frame type (u8) + sequence number (u32)
```

or using `std::mem::sizeof::<Protocol>() + std::mem::sizeof::<FrameType>() + std::mem::sizeof::<SequenceNumber>()` for that matter 😄